magento/magento2
# #26698 Fix Fatal error region getId() on NULL collection in Paypal Nvp API

### Description (*)


To fix the problem `PHP Fatal error: Uncaught Error: Call to a member function getId()` I add a validation on region collection before using `$region->getId()` 

### Fixed Issues (if relevant)

1. magento/magento2#26698: Paypal issue with region on 2.3.4

### Manual testing scenarios (*)

1. Enable paypal w/o review (haven't tried with review)
2. Add product to cart as guest or logged in (doesn't matter)
3. Go to checkout
4. Enter your shipping address to be in UK for example and enter some state, I used Nottinghamshire
5. Place order, confirm from paypal side
6. You will get a success page 

### Questions or comments

N/A
### Contribution checklist (*)
